deploy-manage/deploy/kibana-reporting-configuration.md

```diff
@@ -1,31 +1,281 @@
+::::
+
+```console
+PUT <kibana host>:<port>/api/security/role/custom_reporting_user
+{
+  "elasticsearch": {
+    "cluster": [],
+    "indices": [],
+    "run_as": []
+  },
+  "kibana": [{
+    "spaces": ["*"],
+    "base": [],
+    "feature": {
+      "dashboard_v2": ["generate_report",  <1>
+      "download_csv_report"], <2>
+      "discover_v2": ["generate_report"], <3>
+      "canvas": ["generate_report"], <4>
+      "visualize_v2": ["generate_report"] <5>
+    }
+  }]
+}
+```
+
+1. Grants access to generate PNG and PDF reports in **Dashboard**.
+2. Grants access to generate CSV reports from saved Discover session panels in **Dashboard**.
+3. Grants access to generate CSV reports from saved Discover sessions in **Discover**.
+4. Grants access to generate PDF reports in **Canvas**.
+5. Grants access to generate PNG and PDF reports in **Visualize Library**.
+
+### Grant users access with a Basic license [grant-user-access-basic]
+
+With a Basic license, you can grant users access with custom roles to {{report-features}} with [{{kib}} application privileges](/deploy-manage/users-roles/cluster-or-deployment-auth/kibana-privileges.md). However, with a Basic license, sub-feature privileges are unavailable. [Create a role](#grant-user-access), then select **All** privileges for the applications where users can create reports.
+
+:::{image} /images/kibana-kibana-privileges-with-reporting-basic.png
+:alt: Kibana privileges with Reporting options, Basic license
+:screenshot:
+:::
+
+With a Basic license, sub-feature application privileges are unavailable, but you can use the [role API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-security-put-role) to grant access to CSV {{report-features}}:
+
+```console
+PUT localhost:5601/api/security/role/custom_reporting_user
+{
+  "elasticsearch": { "cluster": [], "indices": [], "run_as": [] },
+  "kibana": [
+    {
+      "base": [],
+      "feature": {
+        "dashboard_v2": [ "all" ], <1>
+        "discover_v2": [ "all" ], <2>
+      },
+      "spaces": [ "*" ]
+    }
+  ],
+  "metadata": {} <3>
+}
+```
+
+1. Grants access to generate CSV reports from saved Discover sessions in **Discover**.
+2. Grants access to generate CSV reports from saved Discover session panels in **Dashboard**.
+3. Optional
+
+### Grant access using an external provider [grant-user-access-external-provider]
+
+If you are using an external identity provider, such as LDAP or Active Directory, you can assign roles to individual users or groups of users. Role mappings are configured in [`config/role_mapping.yml`](/deploy-manage/users-roles/cluster-or-deployment-auth/mapping-users-groups-to-roles.md).
+
+For example, assign the `kibana_admin` and `reporting_user` roles to the Bill Murray user:
+
+```yaml
+kibana_admin:
+  - "cn=Bill Murray,dc=example,dc=com"
+reporting_user:
+  - "cn=Bill Murray,dc=example,dc=com"
+```
+
+## System configuration
+```yaml {applies_to}
+  deployment:
+    self: all
+```
```

Comment:
I know you explained it above, but I'd repeat again 

````suggestion
```
The following configurations are required at {{es}}, {{kib}}, and OS levels to support reporting features.

These steps apply only to **self-managed deployments**. Orchestrated deployments include this configuration by default. For more details on different deployment options, refer to [](/deploy-manage/deploy.md).
````